data-model/SDP-data-model-templates/manuscript-object_sdp-template.json

```diff
@@ -132,11 +220,14 @@
       ]
     }
   ],
+  "other_versions": [
+    "array of URIs"
+  ],
   "iiif": {
     "iiif_manifest": "Manifest URL",
     "text_direction": "CV",
     "viewing_hint": "CV",
-    "range": "???"
+    "thumbnail_url": "IIIF Image URL"
```

Comment:
This is another field from airtable I realized we were missing. I added it to the JSON under the assumption that we will (eventually) use the SMDP as the canonical data form, rather than keeping a separate Airtable version?